src/app/app.module.ts

```diff
@@ -33,3 +43,7 @@ import { getMatPaginatorUkr } from './shared/mat-paginator-config/mat-pagination
 })
 export class AppModule {
 }
+
+export function HttpLoaderFactory(http: HttpClient) {
+  return new TranslateHttpLoader(http);
+}
```

Comment:
Can we create something like HttpLoader.ts  and export  this function? 
And in the TranslateModule use HttpLoaderFactory()

Reply:
here you are probably right however first I want to make it work and then try to refactor this code when I see things in common

Reply:
Okay)